servicelog/appender/logstash.go

```diff
@@ -20,19 +23,24 @@ const (
 )
 
 type logstashConfig struct {
-	Protocol  string `required:"true"`
-	Address   string `required:"true"`
-	RateLimit int    `split_words:"true"`
-	SizeLimit int    `split_words:"true"`
+	Protocol                 string `default:"tcp"`
+	Address                  string
+	DiscoveryRefreshInterval time.Duration `default:"1s" split_words:"true"`
+	DiscoveryServiceName     string        `split_words:"true"`
+	RateLimit                int           `split_words:"true"`
+	SizeLimit                int           `split_words:"true"`
+	TCPKeepAlive             time.Duration `default:"5s" split_words:"true"`
```

Comment:
Have you tested what this parameter will look like? I don't trust split_words when acronyms are involved. ;)

Reply:
I set the variable name directly.

Reply:
@AlfredBroda fixed